apps/registry/src/index.ts

```diff
@@ -22,6 +19,20 @@ async function runHealthCheck(ns: DurableObjectNamespace, version: string) {
 
 export default {
   async fetch(request: Request, env: RegistryDOEnv): Promise<Response> {
+    const url = new URL(request.url);
+
+    if (url.searchParams.get('blockNumber') !== '') {
```

Comment:
Does this require a null check as well in case of no search params or blockNumber is not defined